### PR TITLE
Fix check is upload empty

### PR DESCRIPTION
**Pull request by keatliang2005**

### CODE REVIEW

File: src/Http/Controllers/FilepondController.php [hunk 1]
```diff
@@ -31,6 +31,12 @@ public function upload(Request $request)
     {
         $input = $request->file(config('filepond.input_name'));
         $file = is_array($input) ? $input[0] : $input;
+        
+        if($input == null){
```
Comment (Sopamo) [inline src/Http/Controllers/FilepondController.php:35]: Looks good, could you just use a type safe check?
```suggestion
        if($input === null){
```

Comment (keatliang2005) [reply src/Http/Controllers/FilepondController.php:35]: type safe check as @Sopamo  suggested

Comment (keatliang2005) [reply src/Http/Controllers/FilepondController.php]: thanks for suggestion :)

**Comment by Sopamo** [comment]
Perfect, thanks for the contribution!

**Pull request merged**